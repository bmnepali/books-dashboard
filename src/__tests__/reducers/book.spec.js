import booksReducer from '../../reducers/book';

describe('Books Reducer', () => {
  it('returns the initial state', () => {
    expect(booksReducer(undefined, {})).toMatchSnapshot();
  });
});
