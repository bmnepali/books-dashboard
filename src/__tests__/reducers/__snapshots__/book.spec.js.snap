// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Books Reducer returns the initial state 1`] = `
Object {
  "books": Array [],
  "category": "Hardcover Fiction",
  "error": null,
  "isLoading": false,
}
`;

exports[`Books Reducer returns updated state on loading books 1`] = `
Object {
  "books": Array [],
  "category": "Hardcover Fiction",
  "error": null,
  "isLoading": true,
}
`;
